Basic Select/Weather-Observation-Station-12.sql

```diff
@@ -1,16 +1 @@
-# Author: Thomas George Thomas
-select distinct(city) from station where(
-    city not like "a%" and
-    city not like "e%" and
-    city not like "i%" and
-    city not like "o%" and
-    city not like "u%"   
-   )
-    and
-    (
-         city not like "%a" and
-    city not like "%e" and
-    city not like "%i" and
-    city not like "%o" and
-    city not like "%u"
-    );
+select distinct city from station where left(city,1) not in('a','e','i','o','u') or right(city,1) not in('a','e','i','o','u');
```

Comment:
@anmolkumari Please update 'or' clause to 'and' clause as mentioned in the conversation.